### DISCUSSION TITLE
tree.sync timeout?

### DISCUSSION THREAD

**Discussion by CDWimmer** [Rapptz/discord.py#9544] [Q&A]
Howdy,

I know you're only supposed to run tree.sync sparingly as Discord gets a little upset if you run it too often. Is there any feedback as to the wait time/timeout period until you can sync again? Does the API return this information such that it can be queried or do you just have to try it and see if you get paused for an unknown time. 

Essentially, whilst testing, I'd like to be able to sync as often as possible without getting myself timed out to oblivion. Or to at least know how long oblivion is.

**Comment by Rapptz** [comment]
You're limited to 200 new commands a day. Edits do not count towards this. Generally speaking it's not recommended to spam endpoints regardless. You can set `max_ratelimit_timeout` when initialising a `Client` subclass if you want the library to raise instead of sleep when it hits that threshold. That's about all the library lets you control in terms of rate limit behaviour.
